homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -204,7 +204,8 @@ def update(self):
         if self._destination is not None and self._origin is not None:
             try:
                 params = WazeRouteCalculator.WazeRouteCalculator(
-                    self._origin, self._destination, self._region)
+                    self._origin, self._destination, self._region,
+                    log_lvl="DEBUG")
```

Comment:
Looks like it has to be a valid `logging` level, ie `logging.DEBUG`.

Reply:
This is where the module sets the loglevel: https://github.com/kovacsbalu/WazeRouteCalculator/blob/3384b35aef9cdf58b750a374d0f874c39e162f1c/WazeRouteCalculator/WazeRouteCalculator.py#L43

Is "DEBUG" in my code fine, or don't I understand what you're saying?

Reply:
```py
...
log_lvl=logging.DEBUG)
```

Reply:
Thanks. 

Here I was, thinking I was doing an easy change ;-)

Reply:
Yet, setting it to ```debug``` makes the module spit out everything in the environment

 sudo journalctl -f -u home-assistant@homeassistant
```
Feb 22 21:00:15 RaspberryPi hass[23972]: From: 51.6586303,5.6339602 - to: 51.65861,5.633974
Feb 22 21:00:15 RaspberryPi hass[23972]: From: 52.086694,5.10844 - to: 51.65861,5.633974
Feb 22 21:00:15 RaspberryPi hass[23972]: From: 51.65861,5.633974 - to: 52.086694,5.10844
Feb 22 21:00:15 RaspberryPi hass[23972]: Start coords: (51.6586303, 5.6339602)
Feb 22 21:00:15 RaspberryPi hass[23972]: Start coords: (52.086694, 5.10844)
Feb 22 21:00:15 RaspberryPi hass[23972]: End coords: (51.65861, 5.633974)
Feb 22 21:00:15 RaspberryPi hass[23972]: End coords: (51.65861, 5.633974)
Feb 22 21:00:15 RaspberryPi hass[23972]: Start coords: (51.65861, 5.633974)
Feb 22 21:00:15 RaspberryPi hass[23972]: End coords: (52.086694, 5.10844)
Feb 22 21:00:16 RaspberryPi hass[23972]: 2019-02-22 21:00:16 ERROR (SyncWorker_17) [custom_components.sensor.waze_travel_time] Error on retrieving data: empty response
Feb 22 21:00:16 RaspberryPi hass[23972]: 2019-02-22 21:00:16 ERROR (SyncWorker_9) [custom_components.sensor.waze_travel_time] Error on retrieving data: empty response
Feb 22 21:00:20 RaspberryPi hass[23972]: Time 55.70 - 78.02 minutes, distance 78.61 - 104.36 km.
```

In my home-assistant.log only the last 2 lines show up though (which makes sense, since I have logger set to Warn.

Reply:
If you filter by priority with eg `-p warning` do you get less lines in the journal?

Reply:
Yes 👍